roles/vault/tasks/vault.yml

```diff
@@ -17,14 +17,14 @@
 
 - name: Check if vault is initialized
   uri:
-    url: "https://{{ vault_vip_url }}:8200/v1/sys/init"
+    url: "https://{{ vault_vip_address }}:8200/v1/sys/init"
```

Comment:
I'd say it's not a URL, but an IP or hostname. Is it actually a VIP, or just the address by which you access vault?